docs/guides/intro-to-graphql.md

```diff
@@ -1,9 +1,407 @@
+- `createUser`
+- `createUsers`
+- `updateUser`
+- `updateUsers`
+- `deleteUser`
+- `deleteUsers`
+
+### createUser
+
+Add a single User to the Users list. Requires a `data` parameter that is an object where keys match the field names in the list definition and the values are the data to create.
+
+#### Usage
+
+```gql
+mutation {
+  createUser(data: { name: "Mike" }) {
+    id
+  }
+}
+```
+
+### createUsers
+
+Creates multiple users. Parameters are the same as `createUser` except the data parameter should be an array of objects.
+
+#### Usage
+
+```gql
+mutation {
+  createUsers(data: [{ name: "Mike" }]) {
+    id
+  }
+}
+```
+
+### updateUser
+
+Update a User by ID. Accepts an `id` parameter that should match the id of a user item. The object should contain keys matching the field definition of the list. Only keys that you wish to update need to be provided.
+
+#### Usage
+
+```gql
+mutation {
+  updateUser(id: ID, data: { name: "Simon" }) {
+    id
+  }
+}
+```
+
+### updateUsers
+
+Update multiple Users by ID. Accepts a single data parameter that contains an array of objects. The object parameters are the same as `createUser` and should contain an `id` and nested `data` parameter with the field data.
+
+```gql
+mutation {
+  updateUsers(data: [{ id: ID, data: { name: "Simon" } }]) {
+    id
+  }
+}
+```
+
+### deleteUser
+
+Delete a single Entity by ID. Accepts a single parameter where the `id` matches a user id.
+
+```gql
+mutation {
+  deleteUser(id: ID)
+}
+```
+
+### deleteUsers
+
+Delete multiple entities by ID. Similar to `deleteUser` where the `id` parameter is an array of ids.
+
+```gql
+mutation {
+  deleteUsers(id: [ID])
+}
+```
+
+## Executing Queries and Mutations
+
+Before you begin writing application code, a great place test queries and mutations is the [GraphQL Playground](https://www.apollographql.com/docs/apollo-server/features/graphql-playground/). The default path for Keystone's GraphQl Playground is `http://localhost:3000/admin/graphql`. Here you can execute queries and mutations against the Keystone API without writing any JavaScript.
+
+Once you have determined the correct query or mutation, you can add this to a front-end application. To do this you will need to submit a `POST` request to Keystone's API. The default API endpoint is: `http://localhost:3000/admin/api`.
+
+In our examples we're going to use the browser's [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) for older browsers you may need to polyfill this or use a library like [axios](https://github.com/axios/axios).
+
+First of all let's save a simple query in a variable:
+
+```javascript
+const GET_ALL_USERS = `
+query GetUsers {
+  allUsers {
+    name
+    id
+  }
+}`;
+```
+
+We can then execute this query using a `POST` request:
+
+```javascript
+fetch('/admin/api', {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+  },
+  body: {
+    query: GET_ALL_USERS,
+  },
+}).then(result => result.json());
+```
```

Comment:
Can we combine the above two steps into a single example? There's no technical need to split things out into a separate variable, and it keep the example together + copy-pastable:

```javascript
fetch('/admin/api', {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
  },
  body: {
    query: `
      query GetUsers {
        allUsers {
          name
          id
        }
      }
    `,
  },
}).then(result => result.json());
```

Reply:
Do you think this reads better? I kinda wanted them to think about the query first then the post, then add variables to it later. Building on....

Again not strongly opinionated - I just want something that will be a starting point for people.

Reply:
That's a great point! Perhaps keep them seperate, but combine them into the same code block? Best of both worlds!

Reply:
Cool I'll have slight reword here and see if I can make it flow better